packages/rum-core/src/domain/lifeCycle.ts

```diff
@@ -21,8 +21,7 @@ export enum LifeCycleEventType {
   BEFORE_UNLOAD,
   RAW_RUM_EVENT_COLLECTED,
   RUM_EVENT_COLLECTED,
-  RECORD_STARTED,
-  RECORD_STOPPED,
+  REPLAY_STATS_UPDATED,
```

Comment:
IMO, it doesn't feel like a life cycle event. Former once felt better in that sense. Could we do the same without a life cycle event?

Reply:
It feels a bit weird to me too.

Reply:
Would three events like `REPLAY_SEGMENT_CREATED` `REPLAY_RECORD_ADDED` and `REPLAY_BYTES_WRITTEN` do?

Reply:
I would be in favor of not using Lifecycle events for this use case.
Maybe a stateful stats object used as a dependency by both view and record logic.